src/fr24/livefeed.py

```diff
@@ -159,23 +165,25 @@ def livefeed_response_parse(data: bytes) -> LiveFeedResponse:
 
 def livefeed_playback_response_parse(data: bytes) -> LiveFeedResponse:
     """:param data: raw protobuf message"""
-    lfr = LiveFeedPlaybackResponse()
+    lfr = PlaybackResponse()
     lfr.ParseFromString(data)
     return lfr.live_feed_response
 
 
 def livefeed_flightdata_dict(
-    lfr: LiveFeedResponse.FlightData,
+    lfr: Flight,
 ) -> LiveFeedRecord:
     """Convert the protobuf message to a dictionary."""
+    # NOTE: i've adopted the official protobuf names for consistency
+    # but we do not change the dictionary keys for compatibility
     return {
         "timestamp": lfr.timestamp,
         "flightid": lfr.flightid,
-        "latitude": lfr.latitude,
-        "longitude": lfr.longitude,
-        "heading": lfr.heading,
-        "altitude": lfr.altitude,
-        "ground_speed": lfr.ground_speed,
+        "latitude": lfr.lat,
+        "longitude": lfr.lon,
+        "heading": lfr.track,  # most notably, this might cause confusion
```

Comment:
Please don't add "heading" to the confusion 😭:
ADS-B contains the true track angle; you may have information about the heading angle in some EMS entries.

See [here](https://aviationbook.netlify.app/chapters/data_sources/flight-tracking.html#ads-b) (warning box) for details

Reply:
This was fixed in https://github.com/cathaypacific8747/fr24/pull/24/commits/345b8cdeb99e25e3410f80a9aeda255597e048ed